File: Python/paymentCard.py
from payment import Payment
import datetime

class PaymentCard(Payment):
    id = int
    number = str
    cvv = str
    dateExp = datetime.date

    def __init__(self, id, number, cvv, dateExp):
        super.__init__(id)
        self.number = number
        self.cvv = cvv
        self.dateExp = dateExp

File: Java/PaymentCard.java
import java.util.*;
class PaymentCard extends Payment {
    String number; 
    String cvv; 
    Date dateExp;
    
    public PaymentCard(Integer id, String number, String cvv, Date dateExp){
        super(id);
        this.number = number;
        this.cvv = cvv;
        this.dateExp = dateExp;
    }
}